spec/services/record/update_spec.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
```

Comment:
You have app/services/update_record_service.rb

So spec should be also located as spec/services/update_record_service_spec.rb